libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/filterViews/FilterViewsList.tsx

```diff
@@ -0,0 +1,178 @@
+    filterViews: IDashboardFilterView[] | undefined;
+    onAddNew: () => void;
+    onClose: () => void;
+}
+
+export const FilterViewsList: React.FC<IFilterViewsDropdownBodyProps> = ({
+    filterViews,
+    onAddNew,
+    onClose,
+}) => {
+    const theme = useTheme();
+    const dispatch = useDashboardDispatch();
+    const [filterViewToDelete, setFilterViewToDelete] = React.useState<IDashboardFilterView | undefined>(
+        undefined,
+    );
+
+    return (
+        <>
+            {filterViewToDelete ? (
+                <FilterViewDeleteConfirm
+                    filterView={filterViewToDelete}
+                    onConfirm={() => {
+                        dispatch(deleteFilterView(filterViewToDelete.ref));
+                        setFilterViewToDelete(undefined);
+                    }}
+                    onCancel={() => setFilterViewToDelete(undefined)}
+                />
+            ) : null}
+            <div className="configuration-panel configuration-panel__filter-view__list">
+                <div className="configuration-panel-header">
+                    <Typography tagName="h3" className="configuration-panel-header-title">
+                        <div className="gd-title-with-icon">
+                            <FormattedMessage id="filters.filterViews.dropdown.title" />
+                            <span className="gd-bubble-trigger-wrapper">
+                                <BubbleHoverTrigger>
+                                    <Icon.QuestionMark
+                                        color={theme?.palette?.complementary?.c7 ?? "#B0BECA"}
+                                        width={16}
+                                        height={16}
+                                    />
+                                    <Bubble alignPoints={HEADER_TOOLTIP_ALIGN_POINTS}>
+                                        <div className="gd-filter-view__list__tooltip">
+                                            <FormattedMessage
+                                                id="filters.filterViews.dropdown.tooltip"
+                                                values={{
+                                                    p: (chunks) => <p>{chunks}</p>,
+                                                }}
+                                            />
+                                        </div>
+                                    </Bubble>
+                                </BubbleHoverTrigger>
+                            </span>
+                        </div>
+                    </Typography>
+                </div>
+                <div className="configuration-category">
+                    {filterViews && filterViews.length > 0 ? (
+                        filterViews.map((filterView) => (
+                            <FilterListItem
+                                key={objRefToString(filterView.ref)}
+                                item={filterView}
+                                onApply={() => {
```

Comment:
should not be these handlers defined via useCallbacks?

Reply:
I don't think it is necessary. This way everything would have to be wrapped. Unless it is used some heavy hook or performance is bad, it does not need to be wrapped imho. Also, this is called in `.map` so I would have to move the whole logic down to the component, provide `onClose` to it and leave the component to handle the creation of the callbacks. It look nicer to have it here on the higher layer.